cmd/createcluster.go

```diff
@@ -120,6 +123,11 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		conf.Network = eth2util.Goerli.Name
 	}
 
+	// Ensure sufficient auth tokens are provided for the keymanager addresses
+	if len(conf.KeymanagerAddrs) != len(conf.KeymanagerAuthTokens) {
+		return errors.New("no of keymanager addresses and authentication tokens don't match")
```

Comment:
suggest making it clear that these are misconfigured flags. Suggest `Number of --keymanager-addresses do not match --keymanager-auth-tokens. Please fix configuration flags.`